docs/providers/aws/guide/serverless.yml.md

```diff
@@ -29,6 +29,7 @@ provider:
   name: aws
   runtime: nodejs12.x
   stage: ${opt:stage, 'dev'} # Set the default stage used. Default is dev
+  partition: ${opt:partition, 'aws'} # Overwrite the default partition used. Default is aws
```

Comment:
Once of the problems with this approach is that the user needs to know that they are able to take advantage of the option.